console/src/config.rs

```diff
@@ -28,7 +29,17 @@ pub struct Config {
 
     #[clap(flatten)]
     pub(crate) view_options: ViewOptions,
+
+    /// How long to continue displaying completed tasks and dropped resources
+    /// after they have been closed. Accepted values are:
+    ///
+    /// - durations in the form of 5days 2min 2s.
+    /// - `none` to disable this functionality
+    #[clap(long = "retain-for", default_value = "6s", parse(try_from_str = parse_retain_for))]
+    retain_for: RetainFor,
 }
+#[derive(Debug)]
+struct RetainFor(Option<Duration>);
```

Comment:
nit: add a line of whitespace between struct definitions please :)
```suggestion
}

#[derive(Debug)]
struct RetainFor(Option<Duration>);
```